container-images/tcib/base/os/tempest/run_tempest.sh

```diff
@@ -300,8 +300,8 @@ function run_git_tempest {
     tempest init openshift
     pushd $TEMPEST_DIR
 
-    discover-tempest-config ${TEMPESTCONF_ARGS} ${TEMPESTCONF_OVERRIDES}
-    tempest run ${TEMPEST_ARGS}
+    discover-tempest-config ${TEMPESTCONF_ARGS} ${TEMPESTCONF_OVERRIDES} \
+    && tempest run ${TEMPEST_ARGS}
```

Comment:
technically it could be like this:
`
command $$
command
`
&& doesn't require \ , bash knows another command follows .. but it looked strange to me, so I chose this style to really highlight that the second command depends on the first one being successful :
`
command \
&& command
`